file/file-alfresco/src/test/java/uk/gov/justice/services/file/alfresco/requester/AlfrescoFileRequesterIT.java

```diff
@@ -87,6 +98,50 @@ public void shouldReturnResponseFromAlfresco() throws IOException {
     }
 
     @Test
+    public void shouldReturnPdfContentFromAlfresco() throws IOException {
+        final String fileId = randomUUID().toString();
+        final String mimeType = "application/pdf";
+        final String fileName = "file123.txt";
+
+        restClient = new RestClient();
+
+        headers = new MultivaluedMapImpl<>();
+        headers.add("cppuid", "user1234");
+
+        final Response readMaterialResponseAsPdf = restClient.query(format(basePathWithPort(PORT) + "/%s%s?transformpdf=true", ALFRESCO_WORKSPACE_TRANSFORMATION_PATH, fileId), "*/*", headers);
+
+        assertThat(readMaterialResponseAsPdf.getStatus(), is(OK.getStatusCode()));
+        assertThat(readMaterialResponseAsPdf.getHeaderString(CONTENT_TYPE), CoreMatchers.equalTo("application/pdf;charset=UTF-8"));
```

Comment:
What is the request and these assertions for? It looks like you are testing Alfresco rather than your code?